pytest_watch/tests/test_main.py

```diff
@@ -0,0 +1,20 @@
+import os
+import sys
+import unittest
+
+try:
+    from unittest.mock import patch
+except ImportError:
+    from mock import patch
+
+from pytest_watch import __main__
+from pytest_watch.__main__ import run_cli
+
+
+class TestRunCLI(unittest.TestCase):
```

Comment:
Is there any chance you could switch to classless pytest? Using classes just adds overhead IMHO.

If not, I'll probably go back and do this at a (much) later date.

Reply:
Yes, I will adopt this classless where it is unecessary.